mne/viz/_brain/_brain.py

```diff
@@ -385,7 +388,7 @@ class Brain(object):
        +---------------------------+--------------+---------------+
     """
 
-    def __init__(self, subject_id, hemi, surf, title=None,
+    def __init__(self, subject_id, hemi=None, surf=None, title=None,
```

Comment:
I don't think these should default to None since most of the time people *do* want surfaces. Probably `hemi='both', surfaces='pial'` is the most sensible default?